EIPS/eip-3664.md

```diff
@@ -0,0 +1,314 @@
+---
+eip: 3664
+title: ERC-3664 next generation game NFT
+author: TylerZ(@tylerztl), TY(@nimocat)
+discussions-to: <URL>
+status: Draft
+type: Standards Track
+category: ERC
+created: 2021-07-17
+requires: 20, 1155
+---
+
+## Simple Summary
+ERC-3664 is a standard for next generation game NFTs, which extends ERC-3664.
```

Comment:
I think there's a typo and it should be: "which extends ERC-1155"

Reply:
Thanks for remind, already changed.